sdk/src/utils/test.rs

```diff
@@ -438,6 +438,7 @@ impl crate::signer::RemoteSigner for TempRemoteSigner {
     }
 }
 
+/* This test should be replaced by a rust_native signer if desired to sign from wasm
```

Comment:
Should we even keep it then?
At least flag it with a TO-DO so it's easy to find?